src/test/java/com/googlecode/lanterna/issue/Issue361.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Author Andrey Zelyaev(zella), slightly modified by Andreas(avl42)
+ */
+package com.googlecode.lanterna.issue;
+
+import com.googlecode.lanterna.gui2.*;
+import com.googlecode.lanterna.screen.*;
+import com.googlecode.lanterna.terminal.*;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+public class Issue361 {
+
+    public static void main(String[] args) throws Exception {
+
+        Terminal terminal = new DefaultTerminalFactory().createTerminal();
+        Screen screen = new TerminalScreen(terminal);
+        screen.startScreen();
+
+        BasicWindow window1 = new BasicWindow();
+        window1.setHints(Arrays.asList(Window.Hint.CENTERED));
+
+        BasicWindow window2 = new BasicWindow();
+        window2.setHints(Collections.<Window.Hint>emptyList());
```

Comment:
The original  ```Arrays.asList()``` didn't work with older Java versions.  As of now, Lanterna still works even with Java 1.6 .

Lateron I noticed that  ```Arrays.<Window.Hint>asList()``` would have been simpler and fixed it for older Java, too...

Reply:
nevermind... changed.